frontend/components/admin-user-management/admin-user-management.tsx

```diff
@@ -76,6 +76,32 @@ export default function AdminUserManagement() {
     return <UnauthorisedAccess isLoggedIn={isLoggedIn} />;
   }
 
+  const handleDelete = async (userId: string) => {
+    const token = localStorage.getItem("jwtToken");
+    if (!token) {
+      throw new Error("No authentication token found");
+    }
+
+    try {
+      const response = await fetch(`http://localhost:3001/users/${userId}`, {
+        method: "DELETE",
+        headers: {
+          'Authorization': `Bearer ${token}`,
+        },
+      });
+
+      if (!response.ok) {
+        throw new Error("Failed to delete user");
+      }
+
+      setUsers(users.filter(user => user.id !== userId))
+
+      console.log("User deleted successfully!");
```

Comment:
```suggestion
```
Do we need the logs?

Reply:
Yea, probably don't need the logs and probably the error as well, will just remove them